ldrb/calculus.py

```diff
@@ -199,48 +172,41 @@ def bislerp(
         return Qa
 
     tol = 1e-12
-    qa = rot2quat(Qb)
+    qa = rot2quat(Qa)
     qb = rot2quat(Qb)
 
-    quat_i = np.array([0, 1, 0, 0])
-    quat_j = np.array([0, 0, 1, 0])
-    quat_k = np.array([0, 0, 0, 1])
+    a = qa[0]
+    b = qa[1]
+    c = qa[2]
+    d = qa[3]
+
+    i_qa = np.array([-b, a, -d, c])
+    j_qa = np.array([-c, d, a, -b])
+    k_qa = np.array([-d, -c, b, a])
```

Comment:
If you assume that an array stored as `[a, b, c, d]` corresponds to a quaternion `a+b*i+c*j+d*k`, then this looks correct.